### PR TITLE
python311Packages.mcuuid: init at 1.1.0

### PR DESCRIPTION
**Pull request by clerie**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by NetaliDev — APPROVED** [review]
Result of `nixpkgs-review pr 254444` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.mcuuid</li>
    <li>python310Packages.mcuuid.dist</li>
    <li>python311Packages.mcuuid</li>
    <li>python311Packages.mcuuid.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/mcuuid/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  src = fetchFromGitHub {
+    owner = "clerie";
+    repo = "mcuuid";
+    rev = "refs/tags/${version}";
+    hash = "sha256-YwM7CdZVXpUXKXUzFL3AtoDhekLDIvZ/q8taLsHihNk=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
```
Comment (natsukium) [inline pkgs/development/python-modules/mcuuid/default.nix:21]:
```suggestion
  ];
  
  # upstream has no tests
  doCheck = false;
```
Would you please add `doCheck = false;` explicitly?
> By default the command python setup.py test is run as part of the checkPhase

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#how-to-provide-automatic-tests-to-python-packages-automatic-tests

Comment (clerie) [reply pkgs/development/python-modules/mcuuid/default.nix:21]: done

File: pkgs/development/python-modules/mcuuid/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  doCheck = false;
```
Comment (natsukium) [inline pkgs/development/python-modules/mcuuid/default.nix:23]: Can you add a reason for disabling the test?

Comment (clerie) [reply pkgs/development/python-modules/mcuuid/default.nix:24]: Oh sure, I do

**Review by Janik-Haag — APPROVED** [review]

**Review by natsukium — APPROVED** [review]
LGTM, thanks!

**Pull request merged**